src/shared/components/radar/radar.container.tsx

```diff
@@ -49,7 +54,7 @@ export const Radar = ({ entries, rings, quadrants }) => {
     return R.sortBy(R.prop('position'), radarRings);
   };
 
-  const getQuadrantPosition = (position) => {
+  const getQuadrantPosition = (position: string | number) => {
```

Comment:
there is no type guard here in case when position is a number. 

we should check if it is a number or string or create a default case for switch statement. 
[https://basarat.gitbook.io/typescript/type-system/typeguard](https://basarat.gitbook.io/typescript/type-system/typeguard)

Reply:
Done, changed it